importer_client/python/timesketch_import_client/importer.py

```diff
@@ -490,12 +494,14 @@ def add_excel_file(self, filepath, **kwargs):
 
         self.add_data_frame(data_frame)
 
-    def add_file(self, filepath, delimiter=','):
+    def add_file(self, filepath, delimiter=',', label=''):
         """Add a CSV, JSONL or a PLASO file to the buffer.
 
         Args:
             filepath: the path to the file to add.
             delimiter: if this is a CSV file then a delimiter can be defined.
+            label: optional label of the timeline, used for determining whether
```

Comment:
Ok, yeah this is the data type label. I think I would like to call this something else.. We use `label` for many things and this could potentially be confusing? Let me think about it a bit more

Reply:
renamed to data_label